CMakeLists.txt

```diff
@@ -208,14 +208,34 @@ function(determine_version OUTPUT_VARIABLE)
 
   if(VERSION_MATCH)
     set(${OUTPUT_VARIABLE} ${CMAKE_MATCH_1} PARENT_SCOPE)
+    set(VERSION_STRING ${CMAKE_MATCH_1})
   else()
     message(FATAL_ERROR "Invalid tag detected: ${NEW_VERSION}")
   endif()
+
+  # stuff before + is the version and after the + is the build metadata
+  string(FIND "${VERSION_STRING}" "+" PLUS_POS)
+  string(SUBSTRING "${VERSION_STRING}" 0 ${PLUS_POS} MULTIPASS_SEMANTIC_VERSION)
+  if (PLUS_POS GREATER -1)
+      string(SUBSTRING "${VERSION_STRING}" ${PLUS_POS} -1 MULTIPASS_BUILD_STRING)
+      # if the build metadata starts with a g, the hexadecimal chars after that are a commit hash
+      # otherwise, we do not derive any build string
```

Comment:
Just for my understanding, did you find any valid case with something other than a 'g' after the plus?

Reply:
Yeah, nahhh. It's fine as you have it :) Thanks for clarifying.